### ISSUE TITLE
Error Status code failure 403

### ISSUE THREAD

**Issue by doanquoctoan1998** [AppsFlyerSDK/appsflyer-react-native-plugin#477]
ℹ I tried to install the appsFlyer SDK into my project, but when using logEvent, I received the response 'Status code failure 403.' Please help me check if my devKey has been accepted to use appsFlyer debug. Please provide any issues you see with my configuration. My project urgently needs this SDK to proceed, so please assist me with this matter as soon as possible. Thank you very much.

# Plugin Version
 "react-native-appsflyer": "^6.10.3",
 "react-native": "^0.70.3",
## Report

ℹ AppsFlyer sign-up (account) email: [livestreamtest0621@gmail.com](mailto:livestreamtest0621@gmail.com)

ℹ  Dev key: tYmNbpvWdPFooxGHpAchNR

ℹ framework: react native

ℹ  library: appsflyer-react-native-plugin

ℹ logs: Status code failure 403

ℹ code snippets and any additional relevant information:
```
javascript
const initOptions = {
    isDebug: true,
    devKey: "tYmNbpvWdPFooxGHpAchNR",
    onInstallConversionDataListener: true,
    timeToWaitForATTUserAuthorization: 10,
    onDeepLinkListener: true,
    appId: "",
}

export function AFInit() {
    if (Platform.OS == "ios") {
        appsFlyer.setCurrentDeviceLanguage("EN")
    }
    appsFlyer.setAppInviteOneLinkID("oW4R")
    appsFlyer.initSdk(
        initOptions,
        result => {
            console.log("AppsFlyer initialization succeeded:", result)
        },
        error => {
            console.error("AppsFlyer initialization failed:", error)
        }
    )
    appsFlyer.getAppsFlyerUID((error, appsFlyerUID) => {
        console.log("AppsFlyer ID:", error, appsFlyerUID)
    })
}

export function AFLogEvent(name: string, values: object) {
    appsFlyer.logEvent(
        name,
        values,
        result => {
            console.log("AppsFlyer log succeeded:", result)
        },
        error => {
            console.error("AppsFlyer log failed:", error)
        }
    )
}
```

**Comment by MoritzCooks** [comment]
Did you find any solution for this?

**Comment by trinitiwowka** [comment]
same problem 
<img width="473" alt="image" src="https://github.com/AppsFlyerSDK/appsflyer-react-native-plugin/assets/8225740/86aa8711-bd87-4591-b4d4-6ea1c63fcae3">

**Comment by MoritzCooks** [comment]
Did you check if there's maybe an adblocker that is preventing you from performing tracking calls?

**Comment by mi5ha** [comment]
@yvza Are you on a paid plan? Many people say that 403 happens when you are on a zero plan

**Comment by iskandarzhilmi** [comment]
Adding context for searchers — the root cause for many `Code 50 / 403` reports
across both the React Native and Flutter plugins is the **AppsFlyer Zero plan**
silently blocking SDK ingestion. SKAN postbacks still work on Zero (Apple sends
those server-to-server), which makes it look like the SDK is "almost working" —
installs show up in the SKAN tab but the SDK dashboard stays at 0.

If the dashboard plan page says "Zero plan, Free for life," you'll get this 403
forever regardless of dev key correctness, bundle ID, or any SDK init pattern.
The fix is either upgrading to Growth ($0.07/conversion) or moving to an MMP
with a free SDK tier (Singular has one).

We confirmed via server-side instrumentation: `error_code: 50, error_message:
"Status code failure: 403"`, same trace across every launch attempt.
